bitnami/thanos/README.md

```diff
@@ -20,6 +20,8 @@ Looking to use Thanos in production? Try [VMware Tanzu Application Catalog](http
 
 This chart bootstraps a [Thanos](https://github.com/bitnami/containers/tree/main/bitnami/thanos) deployment on a [Kubernetes](https://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
 
+Bitnami charts can be used with [Kubeapps](https://kubeapps.dev/) for deployment and management of Helm Charts in clusters.
+
```

Comment:
These lines shouldn't appear in the README file. Can you update your main reference and then rebase your branch? I'm not allowed to update it myself, sorry